hy/completer.py

```diff
@@ -27,6 +27,7 @@
 
 import os
 import re
+import six
```

Comment:
We don't need to add six in my opinion. Just add the following snippet to `hy/_compat.py` and import it `from hy._compat import string_types`.

```py
if PY2:
    string_types = basestring,
else:
    string_types = str,
```